system-linux.c

```diff
@@ -2057,6 +2074,9 @@ system_set_ethtool_settings(struct device *dev, struct device_settings *s)
 
 	ecmd.req.cmd = ETHTOOL_SLINKSETTINGS;
 	ioctl(sock_ioctl, SIOCETHTOOL, &ifr);
+
+	if (s->flags & DEV_OPT_EEE)
+		system_set_ethtool_eee_settings(dev, s);
```

Comment:
For consistency, can you move this up where system_set_ethtool_pause is called? Makes it easier to follow the codeflow

Reply:
thanks for the comment, updated as suggested.